LearningMissionSimulation/PlayGroundGavril.cs

```diff
@@ -39,20 +50,100 @@ public  void CreateAccounts(int accountCount)
                         pendingAccountList.Add(account);
                     }
                 }
-             
             }
         }
+
         public void ActivateAccounts()
         {
             foreach (var account  in pendingAccountList )
             {
                 account.Status = Status.Active; 
                 pendingAccountList.Remove(account);
             }
-            
         }
-    }
-}
 
+        public void CreateSubjects(int subjectCount)
+        {
+            Console.WriteLine("******** Started creating subjects  ********\n");
+            int i = 0;
+            while (i < subjectCount)
+            {
+                Subject subject = ObjectGenerator.GenerateSubject();
+                Console.WriteLine("========== Created  Subject {0} =======\n\n{1}\n", i, subject);
+                subjectList.Add(subject);
+                subjectIdList.Add(subject.Id);
+                i++;
+            }
+            Console.WriteLine("========== Generated {0} Subjects  =======\n", subjectCount);
+            Console.WriteLine("******** Finished creating subjects  ********\n");
+        }
 
+        public void CreateModules(int moduleCount)
+        {
+            Console.WriteLine("******** Started creating modules  ********\n");
+            int i = 0;
+            while (i < moduleCount)
+            {
+                Guid subjectId = subjectIdList[AttributeGenerator.random.Next(0, subjectIdList.Count - 1)];
+                Module module = ObjectGenerator.GenerateModule(subjectId);
+                moduleDictionary.Add(module.Id, module);
+                moduleIdList.Add(module.Id);
+                Console.WriteLine("\n========== Created  Module {0} =======\n\n{1}\n", i, module);
+                i++;
+            }
+            Console.WriteLine("========== Generated {0} Modules  =======\n", moduleCount);
+            Console.WriteLine("******** Finished creating modules ********\n");
+        }
 
+        public void AssignModulesToInstructors()
+        {
+            Console.WriteLine("******** Started assigning modules to instructors  ********\n");
+            int i = 0;
+            foreach (var instructor in instructorList)
+            {
+                instructor.ModuleList = GetModuleList();
+                Console.WriteLine("\n========== Generated List Number {0} of the Instructor Number {1} =======\n\n\n", i + 1, i + 1);
+                i++;
+            }
+            Console.WriteLine("========== Generated Lists of the Instructors Module List  =======\n");
+            Console.WriteLine("******** Finished assigning modules to instructors  ********\n");
+        }
+
+        List<Module> GetModuleList()
+        {
+            ISet<Guid> moduleIdSet = new HashSet<Guid>();
+            List<Module> moduleList = new List<Module>();
+            int totalModuleCount = moduleIdList.Count;
```

Comment:
What would happen if totalModuleCount = 0?
Please make sure your method does not crash the application in that case.